odoo/addons/test_pylint/tests/test_pylint.py

```diff
@@ -106,20 +114,23 @@ def test_pylint(self):
                 paths.append(module_path)
 
         options = [
+            '--rcfile=%s' % os.devnull,
             '--disable=all',
             '--enable=%s' % ','.join(self.ENABLED_CODES),
             '--reports=n',
             "--msg-template='{msg} ({msg_id}) at {path}:{line}'",
-            '--load-plugins=pylint.extensions.bad_builtin',
+            '--load-plugins=pylint.extensions.bad_builtin,_odoo_checkers',
             '--bad-functions=%s' % ','.join(self.BAD_FUNCTIONS),
             '--deprecated-modules=%s' % ','.join(self.BAD_MODULES)
         ]
 
+        env = dict(PYTHONPATH=HERE)
```

Comment:
Just a matter of style.
But shouldn't it be `dict(os.environ, PYTHONPATH=HERE)` ?